OpenCorePkg.dsc

```diff
@@ -272,6 +275,22 @@
   OpenCorePkg/Platform/OpenPartitionDxe/PartitionDxe.inf
   OpenCorePkg/Platform/OpenRuntime/OpenRuntime.inf
   OpenCorePkg/Platform/OpenUsbKbDxe/UsbKbDxe.inf
+  OpenCorePkg/Platform/OpenVariableRuntimeDxe/VariableRuntimeDxe.inf {
+    <PcdsFixedAtBuild>
+      gEfiMdeModulePkgTokenSpaceGuid.PcdEmuVariableNvModeEnable|TRUE
+      gEfiMdeModulePkgTokenSpaceGuid.PcdMaxVariableSize|0x10000
+      gEfiMdeModulePkgTokenSpaceGuid.PcdVariableStoreSize|0x10000
```

Comment:
Is not 64K way too little for emulated NVRAM? Thought we increased that for OpenDuet.

Reply:
Microsoft, effectively the gatekeeper, says the min variable store size for SecureBoot should be 120kb (See Page 7 here: https://uefi.org/sites/default/files/resources/UEFI_Plugfest_Security_Microsoft_Fall_2016.pdf). The MacOS requirements presumably then need to be added to this.

For reference, running QueryVariableInfo on my basic Dell inspiron laptop from 2015 shows that it has a 384kb store (393216 bytes). It would be nice to run this on a post-2012 Mac to get a feel for what Apple is doing with those as they apparently went significantly higher than 64kb as from the Trashcan.

Reply:
This is the value in OpenDuet currently, I just copied it. I'm happy to change it to whatever, in both, ofc.

FWIW, maybe of note that we keep far from the whole contents of NVRAM in the file we save and restore, so I suspect this is in practice enough. (Of course, we have to allow for enough space for whatever the OS puts in there while running, even though much/some of it may be discarded on each reboot - i.e. only what is a) saved by `nvramdump` and then b) kept by `LegacySchema` is preserved across reboots.)

Reply:
Let's keep it as is and then increase as needed.